server/trafficrules/src/main/java/com/coffeejug/trafficrules/db/Progress.java

```diff
@@ -0,0 +1,51 @@
+package com.coffeejug.trafficrules.db;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+import javax.persistence.*;
+
+@Entity
+public class Progress {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Integer id;
+
+    private String mockProgress;
+    @JsonIgnore

```

Comment:
Why are we using `@JsonIgnore`?

Reply:
Just in case we will return User object tp front.

Reply:
YAGNI, principle again :)

Reply:
Also, returning the DB instance to the UI isn't the best idea, IMO

Reply:
ok

Reply:
Why returning the DB instance to the UI isn't the best idea ?

Reply:
Because it's a different layer. It's not good to have a coupling of these layers